lnwallet/wallet.go

```diff
@@ -1143,6 +1129,34 @@ func (l *LightningWallet) handleChanPointReady(req *continueContributionMsg) {
 		}
 	}
 
+	// We received accept_channel, update the ChannelContextSigner.
```

Comment:
... with information needed to validate signing operations

Reply:
Done.